pkg/flag/general_flags.go

```diff
@@ -29,25 +29,34 @@ var (
 		Usage:           "Load configuration from the specified path.",
 		DisableInConfig: true,
 	}
+	DisableVersionCheckFlag = Flag{
+		Name:       "disable-version-check",
+		ConfigName: "disable-version-check",
+		Value:      false,
+		Usage:      "Disable Bearer version checking",
```

Comment:
Do you wanna use the `Hide` attribute so that it doesn’t show up?

Reply:
@cfabianski see #892 for use cases and why this is a useful option.

Reply:
@cfabianski are you happy to leave this one visible?